llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -1135,7 +1135,6 @@ void LowerTypeTestsModule::importFunction(
   } else {
```

Comment:
@pcc I believe even if this works it's wrong fix.

We probably should rather check if LTO have selected this definition as prevailing before renaming it into .cfi (or make it internal linkage it it's not)

WDYT?